test/performance/performance.go

```diff
@@ -22,13 +22,23 @@ import (
 
 	pkgTest "knative.dev/pkg/test"
 	"knative.dev/pkg/test/logging"
+	"knative.dev/pkg/test/mako/alerter/slack"
 	"knative.dev/pkg/test/prometheus"
 	"knative.dev/serving/test"
 
 	// Mysteriously required to support GCP auth (required by k8s libs). Apparently just importing it is enough. @_@ side effects @_@. https://github.com/kubernetes/client-go/issues/242
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 )
 
+// SlackUserName is the user name used to send alert to Slack channels.
+const SlackUserName = "Knative Testgrid Robot"
+
+// SlackChannels is a slice of channels we want to alert for performance regressions.
+var SlackChannels = []slack.Channel{{
+	Name:     "performance",
+	Identity: "CBDMABCTF",
```

Comment:
This should come from `config-mako` as well.  This will let us route eventing problems to `#eventing`.

We may also want distinct channels per Serving benchmark (e.g. load-test is `#autoscaling`, but dataplane-probe is `#networking` and deployment probe is `#serving-api`)